.gitignore

```
@@ -3,3 +3,4 @@
 
 __pycache__
 
+sceta-venv
```

add a newline to the bottom of this file

```suggestion
sceta-venv

```